### PR TITLE
APR-2069 Adding outcome column to applications tab

### PR DESCRIPTION
**Pull request by ConorTill**

### CODE REVIEW

File: src/SFA.DAS.RoatpOversight.Domain/ModerationStatus.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SFA.DAS.RoatpOversight.Domain
+{
+    public static class ModerationStatus
+    {
+        public const string Pass = "Pass";
+    }
```
Comment (MarkFCain) [inline src/SFA.DAS.RoatpOversight.Domain/ModerationStatus.cs:10]: Hi, can you update this class to be:   public class ModerationReviewStatus
    {
        public const string Fail = "Fail";
        public const string Pass = "Pass";
    }

File: src/SFA.DAS.RoatpOversight.Domain/GatewayReviewStatus.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SFA.DAS.RoatpOversight.Domain
+{
+    public static class GatewayReviewStatus
+    {
+        public const string Pass = "Pass";
+    }
```
Comment (MarkFCain) [inline src/SFA.DAS.RoatpOversight.Domain/GatewayReviewStatus.cs:10]: Can you update this class to be
public class GatewayReviewStatus
    {
        public const string Fail = "Fail";
        public const string Pass = "Pass";
    }

**Review by MarkFCain — APPROVED** [review]

File: src/SFA.DAS.RoatpOversight.Web/Views/Oversight/Applications.cshtml [hunk 3]
```diff
@@ -56,22 +56,41 @@
...
+                                            @application.OrganisationName
+                                        </a>
+                                    </td>
+                                    <td data-label="UKPRN" class="govuk-table__cell">@application.Ukprn</td>
+                                    <td data-label="Provider route" class="govuk-table__cell">@application.ProviderRoute</td>
+                                    <td data-label="Application reference number" class="govuk-table__cell">@application.ApplicationReferenceNumber</td>
+                                    <td data-label="Application submitted date" class="govuk-table__cell">@application.ApplicationSubmittedDate.ToSfaShortDateString()</td>
+                                    <td data-label="Outcome" class="govuk-table__cell">
+                                        @{
+                                            bool OutcomePass = application.GatewayReviewStatus == GatewayReviewStatus.Pass
```
Comment (chrisfoster76) [inline src/SFA.DAS.RoatpOversight.Web/Views/Oversight/Applications.cshtml:77]: Could we determine the pass/fail status inside a property on the viewmodel, and keep the markup more readable? It'd be unit testable that way, too.

**Review by chrisfoster76 — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.RoatpOversight.Domain/ApplicationDetails.cs [hunk 4]
```diff
@@ -36,5 +37,9 @@ public class ApplicationDetails
         public DateTime? ModerationOutcomeMadeOn { get; set; }
         public string ModeratedBy { get; set; }
         public string ModerationComments { get; set; }
+        public bool OutcomePass => GatewayReviewStatus == Domain.GatewayReviewStatus.Pass
```
Comment (chrisfoster76) [inline src/SFA.DAS.RoatpOversight.Domain/ApplicationDetails.cs:40]: I'm not sure this is used

**Review by chrisfoster76 — CHANGES_REQUESTED** [review]
One comment about a property I think is unused.

**Review by chrisfoster76 — APPROVED** [review]

**Pull request merged**